### PR TITLE
CI/CD: remove unnecessary `opencv` in all orca tests

### PR DESCRIPTION
**Pull request by hamham223**
### 1. Why the change?

As title, remove all the unncessary `opencv` packages.

### CODE REVIEW

File: .github/actions/orca/orca-pytest-automl-py37-spark3-action/nightly-test/action.yml [hunk 1]
```diff
@@ -11,10 +11,6 @@ runs:
         pip install --pre --upgrade bigdl-orca-spark3[automl]
         
         pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} tensorflow==1.15.0
-        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} tensorflow-gan==2.0.0
-        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} tensorflow-probability==0.7.0
-        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} tensorflow-datasets==3.2.0
```
Comment (hamham223) [inline .github/actions/orca/orca-pytest-automl-py37-spark3-action/nightly-test/action.yml:16]: These 3 can be removed

File: .github/actions/orca/orca-pytest-automl-py37-spark3-action/nightly-test/action.yml [hunk 2]
```diff
@@ -11,10 +11,6 @@ runs:
...
-        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} protobuf==3.15.3
```
Comment (hamham223) [inline .github/actions/orca/orca-pytest-automl-py37-spark3-action/nightly-test/action.yml:17]: Not sure, more test needed

File: .github/workflows/PR_validation.yml [hunk 3]
```diff
@@ -458,7 +458,7 @@ jobs:
     - name: Set up maven
       uses: ./.github/actions/maven-setup-action 
     - name: Setup env
-      uses: ./.github/actions/orca/setup-env/setup-rayonspark-tf2-py37
+      uses: ./.github/actions/orca/setup-env/setup-orca-python-py37-basic
```
Comment (hamham223) [inline .github/workflows/PR_validation.yml:461]: cmake is not needed

**Comment by hamham223** [comment]
[Orca-Pytest-Ray-Py37-Spark3](https://github.com/intel-analytics/BigDL/actions/runs/4322247867/jobs/7576253891#logs) mmcv tests requires opencv

File: .github/actions/orca/orca-pytest-automl-py37-spark3-action/nightly-test/action.yml (hunk 2 above)
Comment (hamham223) [reply .github/actions/orca/orca-pytest-automl-py37-spark3-action/nightly-test/action.yml:17]: better to use protobuf==3.19.5

File: .github/actions/orca/orca-pytest-automl-py37-spark3-action/pr-test/action.yml [hunk 4]
```diff
@@ -17,11 +17,8 @@ runs:
         pip install -i https://pypi.org/simple python/orca/src/dist/bigdl_orca*-py3-none-manylinux1_x86_64.whl
         
         pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} -r python/requirements/orca/requirements_automl.txt
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} protobuf==3.19.5
```
Comment (hkvision) [inline .github/actions/orca/orca-pytest-automl-py37-spark3-action/pr-test/action.yml:20]: need to put this into requirements_automl

**Pull request merged**